ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,53 +1,62 @@
 {
   "images" : [
     {
+      "filename" : "40.png",
```

Comment:
I'm now wondering whether or not we should have a convention when it comes to name these assets. If we need to work with a designer, it should be crystal clear to both developers and designers where the icon fits through its name.

**Example:** if the app icons for iPhone are stored in MOXY's cloud in the directory `projectX/app-icons/ios/iphone`, each app icon could be named after something like `20x20@2x.png`.

I'm not exactly sure what should be the approach here. Shall we discuss this topic with our designers?

Reply:
Yes, I think we should. It would be nice to have this standardized.

Reply:
Done!